### PR TITLE
Update README.md: Add RAX75; Increase time to wait for Ethernet connection

### PR DESCRIPTION
**Pull request by ericshu6**
Worked to fix the web UI interface on my RAX75. Thanks!

### CODE REVIEW

**Comment by ericshu6** [comment]
I also changed the code to (supposedly) wait for 60 seconds for an Ethernet connection instead of just 20 seconds. My router didn't boot up in 20 seconds, so I needed to time the command and turning on the router carefully, and I shouldn't have had to do that.

**Review by jclehner — CHANGES_REQUESTED** [review]
Please see requested change for `nmrp.c`.

File: nmrp.c [hunk 1]
```diff
@@ -497,7 +497,7 @@ int nmrp_do(struct nmrpd_args *args)
 		bool unplugged = true;
 		time_t beg = time_monotonic();
 
-		while (!g_interrupted && (time_monotonic() - beg) < 20) {
+		while (!g_interrupted && (time_monotonic() - beg) < 60) {
```
[the remark this answers is not in the document]
Comment (ericshu6) [reply nmrp.c]: Done

**Comment by jclehner** [comment]
Looking good! Thanks!

**Pull request merged**